docs/_docs/windows.md

```diff
@@ -4,7 +4,7 @@ permalink: /docs/windows/
 ---
 
 While Windows is not an officially-supported platform, it can be used to run
-Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update, 
+Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update (1607) or newer, 
```

Comment:
is this relevant? it's another (albeit optional) step and besides, wouldn't this be irrelevant if you `cd ~` anyways?